src/Test/Runner/Log.elm

```diff
@@ -0,0 +1,76 @@
+module Test.Runner.Log exposing (run, runWithOptions)
+
+{-| # Log Runner
+
+Runs a test and outputs its results using `Debug.log`, then calls `Debug.crash`
+if there are any failures.
+
+This is not the prettiest runner, but it is simple and cross-platform. For
+example, you can use it as a crude Node runner like so:
+
+    $ elm-make LogRunnerExample.elm --output=elm.js
+    $ node elm.js
+
+This will log the test results to the console, then exit with exit code 0
+if the tests all passed, and 1 if any failed.
+
+@docs run, runWithOptions
+-}
+
+import Random
+import Test exposing (Test)
+import Test.Runner.String exposing (Summary)
+import String
+
+
+{-| Run the test using the default `Test.Runner.String` options.
+-}
+run : Test -> a -> a
```

Comment:
Seems reasonable! The idea behind this was that this repo should offer some way for you to try out running tests without needing a full-fledged HTML runner or Node runner.

However, I think realistically we can get by having the example do `Debug.log` on its own (in a browser, since we can already run Tasks there), and then let people use the String runner from `elm-repl` if they want to try it out on a CLI.